Lib/test/libregrtest/main.py

```diff
@@ -534,7 +536,13 @@ def _main(self, tests, kwargs):
             self.rerun_failed_tests()
 
         self.finalize()
-        sys.exit(len(self.bad) > 0 or self.interrupted)
+        if self.bad:
+            sys.exit(2)
+        if self.interrupted:
+            sys.exit(3)
```

Comment:
On Linux scripts terminated with `Ctrl-C` return exit code 130 (http://tldp.org/LDP/abs/html/exitcodes.html). Maybe utilize the same exit code here?